README.md

```diff
@@ -176,6 +176,15 @@ struct EdgeIndices32
 
 These index lists enumerate the vertices that are on the edges of the tile. It is helpful to know which vertices are on the edges in order to add skirts to hide cracks between adjacent levels of detail.
 
+## Tiling Scheme and Coordinate System
+
+By default, the data is tiled according to the [Tile Map Service (TMS)](http://wiki.osgeo.org/wiki/Tile_Map_Service_Specification) layout and global-geodetic profile. These defaults can be varied by specifying the `projection` and `scheme`.
+
+Allowed values for the projection are `EPSG:3857` and `EPSG:4326` and for the scheme are `tms` and `slippyMap`.
```

Comment:
Would you mind adding a little more detail here? For example, EPSG:4326 tilesets have 2 tiles at the root, while EPSG:3857 has only one. Also `tms` numbers tiles from the southwest while `slippMap` numbers them from the northwest.

Reply:
I've pushed an update - does this explain the distinction between them adequately?